src/common/pico_stdlib/include/pico/stdlib.h

```diff
@@ -57,6 +57,12 @@ extern "C" {
 #endif
 
 // PICO_CONFIG: PICO_DEFAULT_LED_PIN, Optionally defined a pin that drives a regular LED on the board, default=undefined, group=pico_stdlib
+
+// PICO_CONFIG: PICO_DEFAULT_LED_PIN_PICO_DEFAULT_LED_PIN_INVERTED, 1 if LED is inverted, 0 otherwise, type=int, default=0, group=pico_stdlib
```

Comment:
Haha, I suspect `PICO_DEFAULT_LED_PIN_PICO_DEFAULT_LED_PIN_INVERTED` ought to be just `PICO_DEFAULT_LED_PIN_INVERTED` :stuck_out_tongue_winking_eye:

Reply:
fixed